README.md

```diff
@@ -156,7 +168,13 @@ TESTS
 
         bundle install
 
-* Run tests with
+##### Run tests
+
+* Run tests with local Vagrant based Oracle XE database
+
+        LOCAL_DATABASE=Y rake spec
```

Comment:
`LOCAL_DATABASE` might be confusing name since we may have other local databases. What about USE_VM_DATABASE or maybe something better.